src/plugins/creditor/credNodeStore.js

```diff
@@ -0,0 +1,100 @@
+// @flow
+
+import {type CreditorNode, type NodeUuid, type NodeTagId} from "./creditorNode";
+import {JsonLog} from "../../util/jsonLog";
+
+export type NodeEntry = {|type: "NODE", entry: CreditorNode|};
+export type NodeDetails = {title: string, id: NodeUuid};
```

Comment:
Why do we need this NodeDetails type? Why not just return the CreditorNodes directly in methods that return NodeDetails?